### PR TITLE
Show Filenames in Multishell Title

### PR DESCRIPTION
**Pull request by JakobDev**
With this PR, you can directly see, which File is opened is which tab, if you have multiple files open.

### CODE REVIEW

File: src/main/resources/assets/computercraft/lua/rom/programs/edit.lua [hunk 1]
```diff
@@ -21,6 +21,10 @@ if not fs.exists( sPath ) and not string.find( sPath, "%." ) then
     end
 end
 
+if multishell and settings.get( "edit.multishell_title" ) == true then
```
Comment (SquidDev) [inline src/main/resources/assets/computercraft/lua/rom/programs/edit.lua]: There isn't any need for `== true`. I know you prefer it, but let's keep it consistent with the rest of the code :).

**Comment by SquidDev** [comment]
I wonder if it'd be worth limiting file names to 20 characters or something? Maybe something like:

```lua
local name = fs.getName( path )
if #name > 20 then name = name:sub(1, 17) .. "..." end
if multishell and settings.get( "paint.multishell_title" ) == true then
  multishell.setTitle( multishell.getCurrent(), "paint (" .. name .. ")" )
end
```

This way really long file names won't fill the tab bar, but most file names will still display fine.

**Comment by lupus590** [comment]
Max length as a settings option?

**Comment by Wilma456** [comment]
@lupus590 Whiyh is a good name for this settings? Maybe multishell.max_filename? What do you think?

**Comment by lupus590** [comment]
`multishell.tabTitleMaxLength` (or is underscore the established convention?)

**Comment by thatcraniumguy** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**